containers/home.tsx

```diff
@@ -1,46 +1,51 @@
 import React from 'react';
-import Head from 'next/head'
-import styles from '../styles/Home.module.css'
+import Head from 'next/head';
 
-export default function Home() {
+// Components
```

Comment:
For now, you can skip to modify the homepage, since the layout will change when we started working on the pages